modules/headless_ai/cfgFunctions/set/fn_setStance.sqf

```diff
@@ -6,14 +6,14 @@ params [
     ["_unitStance", "Auto", [""]]
 ];
 
-LOG_1("setstance: %1",_this);
+//LOG_1("setstance: %1",_this);
 
-private _stance = if ((_groupStance isNotEqualTo "Auto") && {_unitStance isEqualTo "Auto"}) then {
+private _stance = if ((_groupStance != "Auto") && {_unitStance == "Auto"}) then {
```

Comment:
Regression in this file.

Reply:
no, case insensitivity check

Reply:
Ok fair enough, didn't realise this made a case difference.